deploy/manifests/dev-env/custom-server-block.yaml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
We should not be duplicating any of these files either. They will surely get out of sync at some point and cause unnecessary maintenance overhead (which is something we are actively trying to minimize).

What we can do is not using `kustomize` for this and simply use bash to concat various files to render the dev-config.yaml file. I think everything can be re-used, except the `deployment` file (because we don't want the ingress-controller container to be running)?
Does that make sense?

Reply:
Makes sense. It's a bummer we can't use `patchesStrategicMerge`.